pylint/constants.py

```diff
@@ -80,8 +80,9 @@ class DeletedMessage(NamedTuple):
     old_names: List[Tuple[str, str]] = []
 
 
-DELETED_MSGID_PREFIXES = [
-    16,  # the PY3K+ checker, see https://github.com/PyCQA/pylint/pull/4942
+DELETED_MSGID_PREFIXES: List[int] = [
```

Comment:
I don't think we need the comment here. What does this tell us a year from now? Just instantiating an empty list and being able to add later to it is fine for now I think.